webapp/api/data/session-data-defaults.test.js

```diff
@@ -0,0 +1,48 @@
+import sessionDataDefaults from "./session-data-defaults"
+
+describe("session-data-defaults", () => {
+  it("should return an object with expected defaults", () => {
+    const defaults = sessionDataDefaults()
+
+    // Check expected keys are present, uses deep-equality for expected objects
+    expect(defaults).toHaveProperty("report-date", {})
+    expect(defaults).toHaveProperty("wreck-find-date", {
+      day: '',
+      month: '',
+      year: ''
+    })
+    expect(defaults).toHaveProperty("personal", {
+      'full-name': '',
+      email: '',
+      'telephone-number': '',
+      'address-line-1': '',
+      'address-line-2': '',
+      'address-town': '',
+      'address-county': '',
+      'address-postcode': ''
+    })
+    expect(defaults).toHaveProperty("location", {
+      'location-standard': {},
+      'location-given': {},
+      'text-location': '',
+      'location-description': ''
+    })
+    expect(defaults).toHaveProperty("property", {})
+    expect(defaults).toHaveProperty("vessel-information", {
+      'vessel-name': '',
+      'vessel-construction-year': '',
+      'vessel-sunk-year': ''
+    })
+    expect(defaults).toHaveProperty("vessel-depth", null)
+    expect(defaults).toHaveProperty("wreck-description", "")
+    expect(defaults).toHaveProperty("salvage-services", "")
+    expect(defaults).toHaveProperty("submittedFiles", [])
+  })
+
+  it("shouldn't return information saved to the defaults object to a later call", () => {
+    const defaults1 = sessionDataDefaults()
+    defaults1.personal["full-name"] = "Alice"
+    const defaults2 = sessionDataDefaults()
+    expect(defaults2.personal["full-name"]).toBe("")
```

Comment:
I'd like to add an additional expect to check the two sessionDataDefaults objects are not the same:

`expect(defaults1 === defaults2).toBe(false)`

Reply:
I've updated this to check recursively in [1aeb63e](https://github.com/mcagov/droits/pull/223/commits/1aeb63e8d9fcc4053482507a5cb89a28280f7eb4)